common/src/main/java/com/microsoft/identity/common/adal/internal/tokensharing/TokenShareUtility.java

```diff
@@ -99,6 +97,39 @@ public TokenShareUtility(@NonNull final String clientId,
         mTokenCache = cache;
     }
 
+    @NonNull
+    private static String getCanonicalEnvironment(@NonNull final String environment) {
```

Comment:
nit: a comment describing what "canonical environment" actually means would be helpful.

Reply:
Refactored out this method and put a `static` method on the enum. The intent should now be clearer

Reply:
48a2ac32baa9d8a894959676e23876143bd7bff6